### ISSUE TITLE
Go OFL

### ISSUE THREAD

**Issue by davelab6** [edwardtufte/et-bembo#2]
Please use the SIL Open Font License.

The OFL is specifically designed for fonts, and the MIT license applied to a font will require the font's copyright notice and the MIT license to be included in all documents that embed the font. 

If you do decide to _go OFL_, please include an OFL.txt file in the repo. Eg https://github.com/impallari/Libre-Bodoni/blob/master/OFL.txt

**Comment by vyp** [comment]
+1.

**Comment by inferno986return** [comment]
I agree completely. SIL OFL is a much better license to go with for font files as it specifically covers embedding.

**Comment by sivapvarma** [comment]
Please use SIL OFL. This is the right license for fonts

**Comment by tobltobs** [comment]
-1
This is one of only a few good quality fonts with a permissive licence. With a SIL OFL licence the option to use this font as base for my own fonts, which I can distribute under my own licence would be gone.

**Comment by rstub** [comment]
@adamschwartz Any chance to have the license adapted? If you want the permissive nature of the MIT license, you could add a "font-embedding clause" as is usual with GPL fonts. Or you could go dual-license: MIT (possibly plus font-embedding clause) and OFL.

**Comment by davelab6** [comment]
@adamschwartz would love to get this updated :)

**Comment by matta** [comment]
> With a SIL OFL licence the option to use this font as base for my own fonts, which I can distribute under my own licence would be gone.

@tobltobs, I do not believe you have this option now, in the sense that you can't simply remove the MIT license from the font.

**Comment by omasanori** [comment]
I truly hope the relicensing would happen.
